token/cli/src/output.rs

```diff
@@ -698,49 +686,230 @@ fn display_ui_extension(
                 Ok(())
             }
         }
+        UiExtension::ConfidentialTransferAccount(UiConfidentialTransferAccount {
+            approved,
+            elgamal_pubkey,
+            pending_balance_lo,
+            pending_balance_hi,
+            available_balance,
+            decryptable_available_balance,
+            allow_confidential_credits,
+            allow_non_confidential_credits,
+            pending_balance_credit_counter,
+            maximum_pending_balance_credit_counter,
+            expected_pending_balance_credit_counter,
+            actual_pending_balance_credit_counter,
+        }) => {
+            writeln!(f, "  {}", style("Confidential transfer:").bold())?;
+            writeln_name_value(f, "    Approved:", &format!("{approved}"))?;
+            writeln_name_value(f, "    Encryption key:", elgamal_pubkey)?;
+            writeln_name_value(f, "    Pending Balance Low:", pending_balance_lo)?;
+            writeln_name_value(f, "    Pending Balance Hi:", pending_balance_hi)?;
```

Comment:
Seems like we should stick to either `lo`/`hi` or `low`/`high`?

Reply:
Ah right, good call, what do you prefer? Maybe we go `Low` / `High` for clarity?

Reply:
Yeah, I think Low/High will be nice.